api/tests/unit/domain/usecases/import-schooling-registrations-from-siecle_test.js

```diff
@@ -35,7 +35,6 @@ describe('Unit | UseCase | import-schooling-registrations-from-siecle', function
     schoolingRegistrationRepositoryStub = {
       addOrUpdateOrganizationSchoolingRegistrations: sinon.stub(),
       addOrUpdateOrganizationAgriSchoolingRegistrations: sinon.stub(),
-      findByOrganizationId: sinon.stub(),
```

Comment:
**question** : je comprends pas pourquoi on enlève ce stub, je le vois toujours utilisé `const existingSchoolingRegistrations = await this.findByOrganizationId({ organizationId });`

Reply:
Bien vu, je croyais qu'on avait droppé ce commit mais visiblement on a oublié.